ros_ws/src/car_control/include/dms_controller.h

```diff
@@ -6,15 +6,17 @@
 #include <std_msgs/Int64.h>
 #include <std_msgs/String.h>
 
-// How often the dead mans switch is checked. in Hz
-constexpr const int DMS_CHECK_RATE = 20;
+constexpr const char* PARAMETER_DMS_CHECK_RATE = "dms_check_rate"; //20
+constexpr const char* PARAMETER_DMS_EXPIRATION = "dms_expiration"; //100
```

Comment:
These comments are very confusing. Please remove them or write something like "Default value is 20Hz". Also doesn't writing the value into the code as a comment defeat the purpose of having them as parameters? If the value is in the code, you need to change it whenever changing the config file.

Reply:
The DMS_CHECK_RATE constant isnt in the code.